scripts/india_census/primary_religion_data/readme.md

```diff
@@ -0,0 +1,101 @@
+# Religion PCA (India & States/UTs - District Level) Overview
+
+
+### Download URL
+It's available at [censusindia.gov.in](https://censusindia.gov.in/2011census/Religion_PCA.html). You can pre-download the dataset and place it in the `data` folder, or you can download it from the URL in real-time. 
+
+### Overview
+The first 4 columns refer to the location for which the data is tabulated. The 5th column is called TRU; it stands for Total, Urban, or Rural. 6th column is the name of the location.
+
+The 7th column is `Religion`, for which the values are defined. The religion column has one of the following values.
+
+- Total
+- Hindu
+- Muslim
+- Christian
+- Sikh
+- Buddhist
+- Jain
+- Other religions and persuasions
+- Religion not stated
+
+
+
+From Column 8 - N, we have values tabulated. The column header is the variable. For example, the 6th column header is `TOT_P`, which has the values for `Total Population`. A list of all such variables and their definition can be found in the CSV  `india_census/common/primary_abstract_data_variables.csv`
+
+ - State - Two-digit state code
+ - District - Three-digit district code inside the specific state
+ - Subdistt -
+ - Town/Village
+ - TRU - Stands for Total, Urban or Rural
+ - Name - Official Name of the place
+ - Religion - Religion for which data is defined
+ - 8- N - Data value columns 
+
+ #### Raw Data Snippet
+
+| State | District | Subdistt | Town/Village | TRU   | Name  | Religion  | TOT_P      | TOT_M     | TOT_F     | P_06      | M_06     | F_06     | 
+|-------|----------|----------|--------------|-------|-------|-----------|------------|-----------|-----------|-----------|----------|----------| 
+| 00    | 000      | 00000    | 000000       | Total | INDIA | Total     | 1210854977 | 623270258 | 587584719 | 164515253 | 85752254 | 78762999 | 
+| 00    | 000      | 00000    | 000000       | Total | INDIA | Hindu     | 966257353  | 498306968 | 467950385 | 127509717 | 66638103 | 60871614 | 
+| 00    | 000      | 00000    | 000000       | Total | INDIA | Muslim    | 172245158  | 88273945  | 83971213  | 28299593  | 14564936 | 13734657 | 
+| 00    | 000      | 00000    | 000000       | Total | INDIA | Christian | 27819588   | 13751031  | 14068557  | 3353497   | 1712933  | 1640564  | 
+
+
+#### Cleaned Data
```

Comment:
It might be worth noting that we've not really defined schema for this, aka created well-defined StatisticalVariable nodes, and have a TODO: to define schema.

Reply:
Added to ` MCFs and Template MCFs` section.